packages/embed-chart/.gitignore

```diff
@@ -0,0 +1 @@
+!build
```

Comment:
I'd add a comment here (or somewhere?) about why we're ignoring this folder and/or why we're putting `index.html` in the `build` folder instead of just at the root.

Reply:
Ya I can put it in the gitignore. It's so the build script in core doesn't need to change at all since eventually we'll want to just remove this and change the build script then. Since there's no build any more, just commit the file directly as it will be published